05-dom-document-loading/2-sortable-table-v1/index.js

```diff
@@ -1,6 +1,107 @@
 export default class SortableTable {
   constructor(headerConfig = [], data = []) {
+	this.headerConfig = headerConfig;
+	this.data = data;
+	this.element = this.createTable();
 
```

Comment:
this.subElements = {
		body: this.element.querySelector('[data-element="body"]')
	  };